b2share/modules/schemas/views.py

```diff
@@ -139,12 +178,125 @@ def get(self, community_schema):
         return community_schema
 
 
+class BlockSchemaListResource(ContentNegotiatedMethodView):
+    view_name = 'block_schema_list'
+
+    def __init__(self, **kwargs):
+        """Constructor."""
+        super(BlockSchemaListResource, self).__init__(
+            method_serializers={
+                'GET': {
+                    'application/json': schemas_list_to_json_serializer,
+                },
+                'POST': {
+                    'application/json': new_schema_to_json_serializer,
+                }
+            },
+            default_method_media_type={
+                'GET': 'application/json',
+                'POST': 'application/json',
+            },
+            default_media_type='application/json',
+            **kwargs
+        )
+
+    def get(self):
+        """Get a list of all schemas."""
+        community_id = request.args['community_id']
+        schemas = BlockSchema.get_all_block_schemas(community_id=community_id)
+        return self.make_response(
+            schemas=schemas,
+            code=200)
+
+    def post(self):
+        """Create a new schema."""
+        data = request.get_json()
+        if data is None:
+            return abort(400)
+
+        schema = BlockSchema.create_block_schema(**data)
+        return self.make_response(
+            schema=schema,
+            code=201,
+        )
+
+
+class BlockSchemaResource(ContentNegotiatedMethodView):
+    view_name = 'block_schema_item'
+
+    def __init__(self, **kwargs):
+        """Constructor."""
+        super(BlockSchemaResource, self).__init__(
+            serializers={
+                'application/json': block_schema_to_json_serializer
+            },
+            default_method_media_type={
+                'PUT': 'application/json'
```

Comment:
There is no need to specify `'PUT': 'application/json'` as the default_media_type is already `application/json`